firefox-ios/Client/Frontend/Browser/Tabs/Views/RemoteTabsTableViewController.swift

```diff
@@ -279,6 +295,67 @@ class RemoteTabsTableViewController: UITableViewController,
         remoteTabsPanel?.remoteTabsClientAndTabsDataSourceDidSelectURL(tab.URL, visitType: VisitType.typed)
     }
 
+    override func tableView(_ tableView: UITableView,
+                            commit editingStyle: UITableViewCell.EditingStyle,
+                            forRowAt indexPath: IndexPath) {
+        if editingStyle == .delete {
+            let clientAndTabs = state.clientAndTabs[indexPath.section]
+            guard let fxaDeviceId = clientAndTabs.client.fxaDeviceId else {
+                // This is a state we probably shouldn't be able to get into
+                // so logging the error
+                logger.log("Error retrieving remote device ID to close tab",
+                           level: .fatal,
+                           category: .tabs)
+                return
+            }
+            let tab = clientAndTabs.tabs[indexPath.item]
+
+            // Setting the two private variables below so that the toast button action function has access to them
+            // since that function cannot have any parameters.
+            self.closeTabRemoteDeviceId = fxaDeviceId
+            self.closeTab = tab
+
+            // Creating a modal with an undo button that will allow the user to undo closing the last remote tab
+            // they attempted to close
+            ActionToast(
+                text: "Tab closed",
```

Comment:
` Tab closed` and `Undo` should be localized.

Reply:
Do you have docs for that process?

Reply:
@lougeniaC64 absolutely: ["How to add and modify Strings"](https://github.com/mozilla-mobile/firefox-ios/wiki/How-to-add-and-modify-Strings)

Reply:
@thatswinnie If I split the string localization changes out into a separate PR is it possible to get it merged by EOD for the string freeze?

Reply:
I created [this branch](https://github.com/mozilla-mobile/firefox-ios/compare/main...lougeniaC64:firefox-ios:close-remote-tabs-strings?expand=1) for the strings which I can turn into a PR if needed. The only thing is I'm not sure what the `tableName` values should be. Is there anywhere that's documented? Thanks!